cumulus/tasks/sf-sns-report/index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const get = require('lodash.get');
+const { StepFunction } = require('@cumulus/ingest/aws');
+const { setGranuleStatus, sns } = require('@cumulus/common/aws');
+const errors = require('@cumulus/common/errors');
+const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
+
+/**
+ * Determines if there was a valid exception in the input message
+ *
+ * @param {Object} event - aws event object
+ * @returns {boolean} true if there was an exception, false otherwise
+ */
+function eventFailed(event) {
+  if (event.exception) {
```

Comment:
Maybe flatten these nested conditionals?
```javascript
if (event.exception && typeof(event.exception) === 'object' && Object.keys(event.exception.length) > 0) {
  return true
}
```

Reply:
yes.  this is neat